src/networkapi.jl

```diff
@@ -183,6 +183,63 @@ function netstoichmat(rn; smap=speciesmap(rn))
 end
 
 
+"""
+    complexstoichmat(rn; smap=speciesmap(rn))
+Returns, 
+Z --> complex composition matrix/complex stoichiometeric matrix
+B --> Incidence matrix
+Δ --> Outgoing matrix
+kd --> Array of rate of reactions
+complexes_mat --> unique intermediate complexes that define the system,
+
+We can define a laplacian matrix from these returned arrays from complexstoichmat as,
+KD = diagm((map(Num, kd)))
+L = B*KD*Δ', and rewrite the ODESystem as x' = -Z*L*exp.(Z'*log.(states(rn)))
+
+and/or also check that, netstoichmat(rn) == Z*B 
+"""
+function complexstoichmat(rn; smap=speciesmap(rn))
```

Comment:
Would it make sense to split this into several functions that individually construct and return some of the other matrices? It seems like this function is doing a lot, and perhaps some of the intermediate matrices would be useful to calculate on their own? You could then make them optional keyword arguments the user can pass in to `complexstoichmat`, using the user's functions if passed in, or otherwise calling those helper functions here.

Reply:
> Would it make sense to split this into several functions that individually construct and return some of the other matrices? It seems like this function is doing a lot, and perhaps some of the intermediate matrices would be useful to calculate on their own? You could then make them optional keyword arguments the user can pass in to `complexstoichmat`, using the user's matrices if passed in, or otherwise calling those helper functions here.

Yes, that makes sense.. I will add individual functions for each entity needed